### ISSUE TITLE
4.0.5: Undo does not work yet

### ISSUE THREAD

**Issue by BjornW** [opto/nostalgy-xpi#225]
I can move a message using the S key and entering a location, press enter and it will be moved to this location. However when pressing CTRL-Z the message will not be retrieved from the location and placed in the old location, so undo does not seem to work yet. 

@opto: Thanks for your hard work! Also, would you please consider using Github for your development process as well? It allows us, well-willing people to participate in the development process. As the saying goes: "Two heads are better than one".

**Comment by opto** [comment]
that is correct. the legacy Version of move gave intermittant functioning, and the New webextention method has not implemented undo yet. I requested that more than a year ago but it is still in an offiziell Bug and not implemented yet.

**Comment by hkclark** [comment]
@opto Could you provide a link to the Bug and we can then all go in and "up vote" it to hopefully get attention soon?  Thanks you!
